submissions/homework_5/cc1759/hw.md

```diff
@@ -0,0 +1,11 @@
+# **Homework Assignment 5: Inserting, Deleting, and Updating Data**
+
+I had to first update my old data to make it comply with what the hw document wanted. I mostly just added more checkouts and profile stuff so I could delete or update them in later sections. I did have the error from auto_incrementing not resetting that you helped me with, but besides that, everything was fine. 
```

Comment:
It didn't have to actually delete any rows. You just needed to make the query to do so. It is good that you made some data to actually see it work though!